Tests/GRPCTests/GRPCAsyncServerHandlerTests.swift

```diff
@@ -304,8 +317,6 @@ class AsyncServerHandlerTests: ServerHandlerTestCaseBase {
     await assertThat(self.recorder.status, .notNil(.hasCode(.unknown)))
   } }
 
-  // TODO: We should be consistent about where we put the tasks... maybe even use a task group to simplify cancellation (unless they both go in the enum state which might be better).
```

Comment:
This `TODO` was dumped here as a note-to-self when we had multiple tasks and an additional state in the handler. It was committed in error, and is unrelated to this PR.